deployment-templates/compute-engine/forseti-instance.py

```diff
@@ -159,7 +159,7 @@ def GenerateConfig(context):
 # Install Forseti Security.
 cd $USER_HOME
 rm -rf *forseti*
-pip install --upgrade pip
```

Comment:
### Temporary Fix upgrade pip using.
```
curl https://bootstrap.pypa.io/get-pip.py | python3
```
Instead of  `pip install -U pip`

For pip2  `pip2 install --upgrade pip`